recipe_scrapers/projectgezond.py

```diff
@@ -0,0 +1,92 @@
+# mypy: allow-untyped-defs
+import re
+
+from ._abstract import AbstractScraper
+
+
+class ProjectGezond(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "projectgezond.nl"
+
+    def author(self):
+        return "Project Gezond"
+
+    def title(self):
+        return self.soup.find("h1", {"class": "entry-title"}).get_text()
+
+    def category(self):
+        return [
+            element.get_text()
+            for element in self.soup.find("span", {"class": "meta-category"}).find_all(
+                "a", {"class": lambda x: x is not None and x.startswith("category")}
+            )
+        ]
+
+    def total_time(self):
+        time_element = self.soup.find("em", string="Bereidingstijd:").parent
+        return "".join(
+            [
+                element.get_text()
+                for element in time_element.children
+                if element.get_text() != "Bereidingstijd:"
+            ]
+        ).strip()
+
+    def yields(self):
+        return re.search(
+            r"\(([^)]+)",
+            self.soup.find(
+                "h2", string=lambda x: x.startswith("Dit heb je nodig")
+            ).get_text(),
+        ).group(1)
```

Comment:
Would this match _any_ text content within parentheses inside the `h2`?

(I'm wondering whether non-yields-related information might be scraped unintentionally from some pages by this code)

Reply:
I changed the regex slightly in my new commit. I now match everything inside the parentheses after 'Dit heb je nodig '. I checked the website and it seems that that's the place where they always put the yields.

Reply:
That change looks good too.  As you suggest, perhaps it's a little bit fragile (may break if & when the site layout/text content changes) -- but those kind of issues can always be resolved over time.